src/main/java/core/basesyntax/model/FruitTransaction.java

```diff
@@ -0,0 +1,65 @@
+package core.basesyntax.model;
+
+import java.util.NoSuchElementException;
```

Comment:
It seems that the import of NoSuchElementException is not used in this class. Remember to keep your imports section clean and remove unused imports to prevent any confusion and keep the codebase neat.